kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -961,6 +1035,21 @@ private Optional<TimePartitioning.Type> parseTimePartitioningType(String rawPart
     }
   }
 
+  private Map<String, String> parseTopic2TableMapConfig(String topic2TableMapString, boolean sanitize) {
+    if (topic2TableMapString.isEmpty()) {
+      return null;
+    }
+    Map<String, String> topic2TableMap = new HashMap<>();
+    // It's already validated, so we can just populate the map
+    for (String str : topic2TableMapString.split(",")) {
+      String[] tt = str.split(":");
+      String topic = tt[0].trim();
+      String table = tt[1].trim();
+      topic2TableMap.put(topic, sanitize ? FieldNameSanitizer.sanitizeName(table) : table);
```

Comment:
do you think after sanitize two table name can be identical? If yes then should we cover during validation

Reply:
That's actually a good point. I also realised that sanitising table name if supplied directly by the user via `topic2table` map doesn't make sense. I removed it and updated the doc